api/dockers/api.go

```diff
@@ -45,7 +45,7 @@ type CreateContainerPayload struct {
 // NewDocker returns a new docker.
 func NewDocker() (*Docker, error) {
 	configAPI := context.GetAPIConfig()
-	dockerHost := fmt.Sprintf("http://%s", configAPI.DockerHostsConfig.Host)
+	dockerHost := fmt.Sprintf("https://%s", configAPI.DockerHostsConfig.Host)
```

Comment:
Fix: I do not know how this was working locally as we are using TLS_VERIFY and certs in Docker API... 🧐